data/xapleaux/xapleaux outfits.txt

```diff
@@ -0,0 +1,80 @@
+
+outfit `Type X "Coleo" Engine`
+	category Engines
+	cost 610000
+	mass 88
+	thumbnail "outfit/type x engine"
+	"X Ring Cradle" -1
+	"engine capacity" -44
+	"outfit space" -44
+	"reverse thrust" 101.8
+	"reverse thrusting energy" 12.8
+	"reverse thrusting heat" 17.9
+	thrust 101.8
+	"thrusting energy" 12.8
+	"thrusting heat" 17.9
+	turn 5263.9
+	"turning energy" 12.8
+	"turning heat" 17.9
+	"flare sprite" "effect/coleo flare"
+		"frame rate" 10
+	"reverse flare sprite" "effect/coleo flare"
+		"frame rate" 10
+	"steering flare sprite" "effect/coleo flare"
+		"frame rate" 10
+	"flare sound" ""
+	"reverse flare sound" ""
+	"steering flare sound" ""
+	description "This thruster pair is one of the three primary drive types produced by the Xapleaux. Unlike the much larger Lepido or efficient but undersized Haltere engines the Coleo produces comparatively more heat and consumes rather more energy to achieve higher thrust outputs per tonne. They are built to propel their Ascomy warships quickly to a point of crisis."
+	description "	Like all Xapleaux engines which are mounted on the long side spars away from the center of mass; the reversible, turbine-style exhaust facilitates reverse thrust just as efficiently and doubles as steering by using opposing thrust on either side. Only a ship with a specialized X Ring Cradle can mount this engine."
+
+outfit `Type I "Lepido" Engine`
+	category Engines
+	cost 488000
+	mass 178
+	thumbnail "outfit/type x engine"
+	"I Ring Cradle" -1
+	"engine capacity" -54
+	"outfit space" -57
+	"reverse thrust" 74.2
+	"reverse thrusting energy" 4.1
+	"reverse thrusting heat" 3.8
+	thrust 74.2
+	"thrusting energy" 4.1
+	"thrusting heat" 3.6
+	turn 3835.8
+	"turning energy" 4.1
+	"turning heat" 3.8
+	"flare sprite" "effect/lepido flare"
+		"frame rate" 10
+	"reverse flare sprite" "effect/lepido flare"
+		"frame rate" 10
+	"steering flare sprite" "effect/lepido flare"
+		"frame rate" 10
+	"flare sound" ""
+	"reverse flare sound" ""
+	"steering flare sound" ""
+	description "This thruster pair is one of the three primary drive types produced by the Xapleaux. Unlike the powerful Coleo or space efficient Haltere engines the Lepido is incredibly massive and produces very little thrust per tonne. However, this 'caterpillar' style drive uses its much more significant, multi-staged turbine length to produce thrust extremely efficiently with low power requirements and little waste heat. For this reason it is popular on civilian ships that can afford to run it on solar power alone."
```

Comment:
It's a reference to "The Hunt for Red October", essentially it's a big, _quiet_, low-output but efficient drive.

It may or may not operate on a particle version of this (who's to say?):
https://www.wired.com/story/the-magnetohydrodynamic-drive-is-realand-you-can-build-one/

Reply:
It's kind of an esoteric reference, and to someone not familiar with "The Hunt for Red October" it's unclear what it means. Perhaps consider changing it to something that most people will recognize as a 'big, quiet, low-output but efficient drive'?
As for the '-', I'm fairly certain it should be there, but I'm not 100% sure.

Reply:
Not sure on the hyphen myself - Googling reveals this is some pretty case-specific stuff. I'd say throw it in since it helps clarify meaning. Also, `'caterpillar'` needs double quotation marks. Here you go:
```suggestion
	description `This thruster pair is one of the three primary drive types produced by the Xapleaux. Unlike the powerful Coleo or space efficient Haltere engines the Lepido is incredibly massive and produces very little thrust per tonne. However, this "caterpillar"-style drive uses its much more significant, multi-staged turbine length to produce thrust extremely efficiently with low power requirements and little waste heat. For this reason it is popular on civilian ships that can afford to run it on solar power alone.`
```

Reply:
> It's kind of an esoteric reference

Yes, sometimes that's intentional.
If you really want to make a case for an alternative I'm happy to consider it, but I've no intention of looking for an alternative myself.